src/main/java/com/corpo/coliseum/domain/service/impl/CorporationServiceImpl.java

```diff
@@ -46,14 +49,15 @@ public void remove(String name, String sport) {
 
     @Override
     public void register(String name, String mail) throws UserException {
-        if(userService.userAlreadySignIn(mail)){
+        if(Boolean.TRUE.equals(userService.userAlreadySignIn(mail))){
```

Comment:
why does the userAlreadySignIn() method return a Boolean which is not primitive (boolean)?
if that changes nothing so much return a primitive type to avoid this kind of test

Reply:
Not sure about if we have to do it this way, but in this case, why just not try userService.userAlreadySignIn(mail) in condition instead of using Boolean.equals ?

Reply:
just more conscise, effectively not mandatory